UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -82,7 +81,7 @@ private void Init()
 			MasterVolume(1f);
 		}
 
-		layerMask = LayerMask.GetMask("Walls", "Door Closed");
+		layerMask = LayerMask.GetMask("Door Closed");
```

Comment:
Codacy found an issue: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6319358)